client/src/js/components/bhEmployeeFinder.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * The employee finder component
+ *
+ * @module components/bhEmployeeFinder
+ *
+ * @description a component to deal with employee, it let you choose a employee
+ * 
+ **/
+ 
+//An employee finder component controller
+ 
+function employeeFinderController (employeeService){ 	
+ 	var ctrl = this;
+
+ 	ctrl.searchOptions = [
```

Comment:
It would be useful to cache the search option using `AppCache`.  [Here is an example](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/js/directives/bhFindPatient.js#L155).

@sfount suggested that we use a JSON object for the `options` array, since it is more readable.  You can read his full explanation in [this comment](https://github.com/IMA-WorldHealth/bhima-2.X/pull/116#discussion-diff-53319845).  It's a very thorough explanation.